acme/acme/client.py

```diff
@@ -56,16 +58,20 @@ class Client(object):  # pylint: disable=too-many-instance-attributes
 
     """
 
-    def __init__(self, directory, key, alg=jose.RS256, verify_ssl=True,
-                 net=None):
+    # pylint: disable=too-many-arguments
```

Comment:
My understanding is putting this comment here disables this check for the rest of the class. Instead, we should put it below the docstring in `__init__` for the same effect to help warn us when classes and functions might be getting too complicated in the future.